docs/chainOps/Upgrades.md

```diff
@@ -0,0 +1,74 @@
+# Sifnode Upgrades
+
+## Packaging and proposing an upgrade
+
+Upgrades on the Sifnodes themselves are automated, however there are several actions required to be performed prior.
+
+1. Generate a new release [here](https://github.com/Sifchain/sifnode/releases).
+
+2. Github actions will then build a new sifnoded binary, add it to a zip file, and upload it to [S3](https://s3.console.aws.amazon.com/s3/buckets/finance.sifchain.release?region=us-west-2&tab=objects).
+
+The parent folder's name will be the SHA256 checksum of the binary. This will be required when broadcasting a proposal to the network. 
+
+Also, get the full S3 URL by clicking on the binary itself (inside the S3 console).  
+
+3. Submit an upgrade proposal to the network:
+
+```
+sifnodecli tx gov submit-proposal software-upgrade <upgrade_name> \
+    --from <from> \
+    --deposit <deposit> \
+    --upgrade-height <height> \
+    --info '{"binaries":{"linux/amd64":"<url>"}}' \
+    --title <title> \
+    --description <description>
+```
+
+Where:
+
+| Parameter | Description |
+|-----------|-------------|
+| `<upgrade_name>` | Name of the upgrade. This must match the upgrade name being used by the upgrade handler in the new binary. |
+| `<from>` | The moniker of the validator proposing the upgrade. |
+| `<deposit>` | The deposit/fee for proposing the upgrade. |
```

Comment:
I think we can mention the deposit required  is configurable thrrough the genesis